students/HABTAMU/session04/copy_src_dst.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python3
+
+# Write a program which copies a file from a source, to a destination (without using shutil, or the OS copy command).
+
+def copy_src2dst():
+    with open('my_src.txt', 'rb') as infile, open('my_dest.txt', 'wb') as outfile:
```

Comment:
nice use of double context manager!

how would you do it if the file was too big for memory?